### PR TITLE
Bug fix for W/Z gen and simple notebooks for gen plotting

### PR DESCRIPTION
**Pull request by kdlong**
* Apply nominal weight for W/Z distributions
* Some gen plotting notebooks
* Some additional plotting tweaks

### CODE REVIEW

File: wremnants/theory_tools.py [hunk 1]
```diff
@@ -93,10 +93,13 @@ def qcdScaleNames():
...
     return ["_".join(["QCDscale", s]) if s != "" else s for s in shifts]
 
-def massWeightNames(matches=None):
+def massWeightNames(matches=None, wlike=False):
     central=10
     nweights=21
     names = [f"massShift{int(abs(central-i)*10)}MeV{'Down' if i < central else 'Up'}" for i in range(nweights)]
+    if wlike:
+        # These are the Z width variations
+        names.extend(["", ""])
```
Comment (bendavid) [inline wremnants/theory_tools.py:102]: Why are these added as empty strings?

File: wremnants/theory_tools.py [hunk 2]
```diff
@@ -93,10 +93,13 @@ def qcdScaleNames():
...
     # If name is "" it won't be stored
```
Comment (kdlong) [inline wremnants/theory_tools.py:103]: It explains it right here :)

File: wremnants/theory_tools.py (hunk 1 above)
Comment (kdlong) [reply wremnants/theory_tools.py:102]: See the comment right below this

Comment (kdlong) [reply wremnants/theory_tools.py:102]: The line right below this tries to explain it. Ultimately each variation gets projected, those with empty names don't get stored in the output root file.
